app/scripts/app.js

```
@@ -337,6 +336,61 @@
   $rootScope.forceFullscreen = false;
 });
 
+function isIp(host) {
+  return (
+    /^(\d{1,3}\.){3}\d{1,3}$/.test(host) &&
+    host.split('.').every(num => parseInt(num) >= 0 && parseInt(num) <= 255)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)